README.rst

```diff
@@ -33,140 +106,69 @@ bpython-urwid
-	parameters for any function you call. The inspect module is tried first,
-	which works with any Python function, and then pydoc if that fails, which
-	seems to be pretty adequate, but obviously in some cases it's simply not
-	possible. I used pyparsing to cure my nested parentheses woes; again, it
-	was nice and easy.
-
-* Rewind.
-	I didn't call this "Undo" because I thought that would be misleading, but
-	"Rewind" is probably as bad. The idea is that the code entered is kept in
-	memory and when the Rewind function is called, the last line is popped and
-	the entire code is re-evaluated. As you can imagine, this has a lot of
-	potential problems, but for defining classes and functions, I've found it
-	to be nothing but useful.
-
-* Pastebin code/write to file.
-	I don't really use the save thing much, but the pastebin thing's great. Hit
-	a key and what you see on the screen will be sent to a pastebin and a URL
-	is returned for you to do what you like with. I've hardcoded
-	paste.pocoo.org in for now, that needs to be fixed so it's configurable.
-	Next release, I promise.
-
-* Flush curses screen to stdout.
-	A featurette, perhaps, but I thought it was worth noting.  I can't
-	personally recall a curses app that does this, perhaps it's often not
-	useful, but when you quit bpython, the screen data will be flushed to
-	stdout, so it basically looks the same as if you had quit the vanilla
-	interpreter.
-
-Configuration
-=============
-See the sample-config file for a list of available options.  You should save
-your config file as ~/.config/bpython/config (i.e
-$XDG_CONFIG_HOME/bpython/config) or specify at the command line::
-
-	bpython --config /path/to/bpython/config
-
-Known Bugs
-==========
-For known bugs please see bpython's issue tracker at github:
-
-http://github.com/bpython/bpython/issues/
-
+===================
 CLI Windows Support
 ===================
 
 Dependencies
 ------------
-Curses
-    Use the appropriate version compiled by Christoph Gohlke
-    http://www.lfd.uci.edu/~gohlke/pythonlibs/
+`Curses`_ Use the appropriate version compiled by Christoph Gohlke.
 
-pyreadline
-    Use the version in the cheeseshop
-    http://pypi.python.org/pypi/pyreadline/
+`pyreadline`_ Use the version in the cheeseshop.
 
+===========
 Recommended
------------
+===========
 Obtain the less program from GnuUtils. This makes the pager work as intended.
 It can be obtained from cygwin or GnuWin32 or msys
 
+==============================
 Current version is tested with
-------------------------------
- * Curses 2.2
- * pyreadline 1.7
+==============================
+* Curses 2.2
+* pyreadline 1.7
 
+============
 Curses Notes
-------------
-The curses used has a bug where the colours are displayed incorrectly:
- * red  is swapped with blue
- * cyan is swapped with yellow
+============
+The curses used has a bug where the colours are displayed incorrectly: * red  is
+swapped with blue * cyan is swapped with yellow
```

Comment:
Will do - this was a typo from reformatting the doc to wrap at 80 chars. I thought I caught them all. I'll change this tonight (unfortunately, real work beckons).